libs/ledger-live-common/src/wallet-api/Exchange/server.ts

```diff
@@ -280,7 +290,7 @@ export const handlers = ({
               binaryPayload: params.hexBinaryPayload,
               exchange,
               feesStrategy: params.feeStrategy,
-              swapId: params.exchangeType === "SWAP" ? params.swapId : undefined,
+              quoteId: quoteOrSwapId,
```

Comment:
you renamed swapId to quoteId but everywhere else you're still using swapId and you have this todo here
https://github.com/LedgerHQ/ledger-live/pull/7696/files#diff-af77015a1ecc0bc635f1da1c8289441144dd58c96c4535215b6c78920618676fR159